src/components/project-details/project-details.module.css

```diff
@@ -3,6 +3,45 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+#project_details {
+  background-color: #ebf5fa;
+  box-shadow: 5px 5px 1px #afb3e0;
+}
+
+.item_box {
+  border-radius: 10px;
+  padding-top: 10px;
+  padding-left: 20px;
+  padding-bottom: 10px;
+  margin-left: 20%;
+  margin-right: 20%;
+  background-color: #f7f7f7;
+  box-shadow: 5px 5px 1px #999999;
+}
```

Comment:
Let's move this styling definition into the top-level index.css file so that it can be used in all areas of the application.